generators/app/index.js

```diff
@@ -258,7 +258,7 @@ module.exports = generators.Base.extend({
           this.disableLogger = answers.disableLogger ? answers.disableLogger :  this.options.disableLogger || 'false';
           this.noMinify = answers.noMinify ? answers.noMinify :  this.options.noMinify || 'false';
           this.enableValidator = this.options.enableValidator || 'false';
-          this.htmlValidator = this.options.enableValidator || '{validator: "http://html5.validator.nu", format: "text", suppressWarnings: true}';
+          this.htmlValidator = this.options.enableValidator || '{"validator": "http://html5.validator.nu", "format": "text", "suppressWarnings": true}';
```

Comment:
hmm that was super out of date...

make sure you audit all the default values to make sure they match the defaults that are in here: https://github.com/rooseveltframework/roosevelt/blob/master/lib/sourceParams.js